configure.ac

```diff
@@ -44,6 +44,8 @@ AC_CONFIG_HEADERS([zfs_config.h], [
 	(mv zfs_config.h zfs_config.h.tmp &&
 	awk -f ${ac_srcdir}/config/config.awk zfs_config.h.tmp >zfs_config.h &&
 	rm zfs_config.h.tmp) || exit 1])
+AC_CACHE_VAL([lt_cv_sys_lib_dlsearch_path_spec],[
```

Comment:
> /sbin/zdb: error while loading shared libraries: libnvpair.so.1: cannot open shared object file: No such file or directory

And this of course won't work: RPATH="/lib64" is the only thing that is making ZFS binaries work where "/lib64" is not in `ld` search path, taking this away allow us to compile but then we can't run the result.